spec/output_formats/dir_spec.rb

```diff
@@ -1,8 +1,45 @@
 require 'spec_helper'
 require 'ronin/recon/output_formats/dir'
+require 'tmpdir'
 
 describe Ronin::Recon::OutputFormats::Dir do
+  subject { described_class.new(path) }
+
+  let(:path) { Dir.mktmpdir }
+
   it "must inherit from Ronin::Core::OutputFormats::OutputDir" do
     expect(described_class).to be < Ronin::Core::OutputFormats::OutputDir
   end
+
+  describe "#<<" do
+    context 'for valid value' do
```

Comment:
Need to add specs for the different `Values::` classes to ensure that they get written to the appropriate file within the output directory.